apps/climatemappedafrica/src/pages/[[...slug]].js

```diff
@@ -3,10 +3,12 @@ import { SWRConfig } from "swr";
 
```

Comment:
Should we name this file `[[...slugs]].js` ?

Reply:
Not sure I follow Mr. @koechkevin; isn't it named `[[...slug]].js` already?

Reply:
I'm just asking @kilemensi we've named slugs in other apps(charterafrica, codeforafrica etc) so I thought we should do the same as well here.

Reply:
Oh `slugs` vs `slug`! I didn't even see that... I didn't create this file so I'm OK with what you guys think is best.

Next.js seem to be using `slug` [though](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments).